data_diff/databases/mysql.py

```diff
@@ -148,3 +148,11 @@ def create_connection(self):
             elif e.errno == mysql.errorcode.ER_BAD_DB_ERROR:
                 raise ConnectError("Database does not exist") from e
             raise ConnectError(*e.args) from e
+
+    def _query_in_worker(self, sql_code: Union[str, ThreadLocalInterpreter]):
```

Comment:
You defined this private method here, but where are you calling it?

Reply:
When `Database` calling `query` function,  `_query` func in `ThreadedDatabase` class will be called. Then `_query_in_worker` in `ThreadedDatabase` will be executed. I put this func here to execute `Mysql`.`_query_in_worker` instead of `_query_in_worker` in `ThreadedDatabase`. If there is any problem, please let me know, thank you

Reply:
Got it, thanks for explaining that and realizing that it does override the inherited class function with the MySQL-specific implementation.